File: incidents-parking/services/crash-incident-go/data/model/verifyIncidents.go
package model

import "go.mongodb.org/mongo-driver/bson/primitive"

const (
	Continue = int(1)
	Finish   = int(2)
	Skip     = int(3)
)

type VerifyIncident struct {
	ID         primitive.ObjectID `bson:"_id,omitempty" json:"_id,omitempty"`
	UserID     string             `bson:"user_id" json:"user_id"`
	IncidentID primitive.ObjectID `bson:"incident_id" json:"incident_id"`
	Option     int                `bson:"option" json:"option"`
	CreateTime int64              `bson:"created_at" json:"created_at"`
	UpdateTime int64              `bson:"updated_at" json:"updated_at"`
}

File: incidents-parking/old/crashsaver-core/src/modules/verifyIncident/domain/model/verifyIncidents.ts
import { ObjectId } from "bson";

/* eslint-disable camelcase */
//1=continue, 2=finish, 3=skip
type Option = 1 | 2 | 3;

export interface CreateVerifyIncident {
  incident_id: string;
  option: Option;
}

export interface VerifyIncident {
  incident_id: string;
  user_id: string;
  option: Option;
  created_time: number;
  updated_time: number;
}